docs/pages/api-docs/image-list-item.json

```diff
@@ -1,22 +1,18 @@
 {
   "props": {
     "children": { "type": { "name": "node" } },
-    "classes": { "type": { "name": "object" } },
     "cols": { "type": { "name": "number" }, "default": "1" },
     "component": { "type": { "name": "elementType" } },
-    "rows": { "type": { "name": "number" }, "default": "1" }
+    "rows": { "type": { "name": "number" }, "default": "1" },
+    "sx": { "type": { "name": "object" } }
   },
   "name": "ImageListItem",
-  "styles": {
-    "classes": ["root", "img", "standard", "woven"],
-    "globalClasses": {},
-    "name": "MuiImageListItem"
-  },
+  "styles": { "classes": [], "globalClasses": {}, "name": null },
```

Comment:
I will need to see what is the issue with the `buildAPI` for removing these. I suppose is the way of how the `ImageListItemClassKey` are being declared.